programs/piaware/faup1090.tcl

```diff
@@ -406,7 +406,7 @@ proc stop_faup1090_close_faup1090_socket_and_reopen {} {
 # attempt_dump1090_restart - restart dump1090 if we can figure out how to
 #
 proc attempt_dump1090_restart {{action restart}} {
-	set scripts [glob -nocomplain /etc/init.d/*dump1090*.sh]
+	set scripts [glob -nocomplain /etc/init.d/*dump1090*]
```

Comment:
Ah okay. Well as of right now this particular version of dump1090 doesn't seem to generate a init.d/dump1090.sh, and therefore breaks our ability to restart/start/stop since the file doesn't exist. What's the right way to fix this? Should building/installing the debian package for dump1090 generate a .sh file?